system-configuration/src/network_configuration.rs

```diff
@@ -0,0 +1,439 @@
+use std::fmt;
+use std::ptr;
+use std::mem;
+use std::net::IpAddr;
+
+use core_foundation_sys::base::kCFAllocatorDefault;
```

Comment:
The `core-foundation` crate re-exports the corresponding `-sys` modules. So you don't need to depend directly on `core-foundation-sys`. You could import it as `use core_foundation::base::kCFAllocatorDefault;`

Reply:
got it.